src/components/Pagination.astro

```diff
@@ -49,6 +49,7 @@ const collapsedPages = collapseRange(page, pages)
 					) : (
 						<a
 							href={link.href}
+							rel="prefetch"
```

Comment:
let's skip this one for now, scrolling to the bottom of the page ends up triggering quite a few prefetches for prev/next + individual pages + terms/privacy

I do like the next/prev arrows though!  Those are probably a lot more likely to be clicked than a page number 

<img width="1048" alt="Screenshot 2023-03-10 at 12 41 29 PM" src="https://user-images.githubusercontent.com/15836226/224398878-c6d98131-2893-4566-8e8d-8abc2ea79a73.png">